src/main/java/edu/harvard/iq/dataverse/api/Dataverses.java

```diff
@@ -342,15 +343,35 @@ public Response setMetadataRoot( @PathParam("identifier")String dvIdtf, String b
     
     @GET
     @Path("{identifier}/facets/")
+    /**
+     * return list of facets for the dataverse with alias `dvIdtf`
+     */
     public Response listFacets( @PathParam("identifier") String dvIdtf ) {
-        return allowCors(response( req -> ok(
-                        execCommand(new ListFacetsCommand(req, findDataverseOrDie(dvIdtf)) )
```

Comment:
The removal of the `ListFacetsCommand` makes me wonder if the permissions at https://github.com/IQSS/dataverse/blob/v4.6/src/main/java/edu/harvard/iq/dataverse/engine/command/impl/ListFacetsCommand.java#L37 should be included. It looks like they checked if the dataverse is published or not.